modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -124,3 +124,98 @@ This command ensures that Runtime Fabric creates the necessary overrides for the
 The `rtfctl uninstall` command deletes any pull secret you specify during installation.
 
 When using `rtfctl` to perform a backup and restore procedure, the backups point to your local registry URL and pull secret. 
+
+== Configure Your Local Registry for Runtime Fabric using Helm
+
+To configure a local registry, you must gather and add the necessary credentials to synchronize the registries. 
+
+=== Before You Begin
+
+Ensure that you've performed the following tasks: 
+
+. Set up, configure, and test your private Docker image registry.
+. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
++
+See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version. 
++
+[NOTE]
+Dependency versions are specific to the Runtime Fabric version.
+
+=== Configure Your Local Registry and Install Runtime Fabric
+To configure a local registry for Runtime Fabric using Helm, follow these steps:
+
+. In your shell script, run the following script to synchronize container images to a local registry. Note that you must update `PLATFORM_URL`, `AGENT_VERSION`,  `AUTHORIZATION_TOKEN`, and `ACTIVATION_DATA`.
```

Comment:
This is generic, not only for the helm usage.

We should use this script as a way to automate what is being defined above as a manual repetitive step.